### ISSUE TITLE
Set private status about User (not logged)

### ISSUE THREAD

**Issue by camlafit** [go-gitea/gitea#1780]
Follow #714 issue

If any user are set in private organization, shouldn't displayed in **explore/users** 
User should be set as private, actually looks possible to hide his email.

Should be nice to have : 
* an option to set default account visibility  (public, private, connected)
* an option on each user profil to ovveride this behavior
* display user data following organization properties.

**Comment by DblK** [comment]
Once #1763 is merged, it can be done easily since *organization* are *user* in database.
1. Add a new ini parameter and reflect in the UI
2. Like organization, allow modification in settings
3. Hide organizations for other members based on organization visibility and current user organizations
4. Modify the explore/users to display according to visibibility

Hope I understand it well.

**Comment by camlafit** [comment]
Hi

Yes idea is very similar. Purpose is to hide contributor information as organization/repo done in #1763.

If we can manage organization visibility, looks logical to have same behavior with user visibility. 
Thanks

**Comment by appIeboy** [comment]
<p align=right><sub><sup><sub><sup><sub><sup><sub><sup><sub><sup><sub><sup><sub><sup><sub><sup><sub><sup><sub><sup><sub><sup><sub><sup><sub><sup><sub><sup><sub><sup><sub><sup>#1798 @lunny @tboerger @sapk @cez81 @geek1011 @lafriks @ethantkoenig @andreynering @DblK </sup></sub></sup></sub></sup></sub></sup></sub></sup></sub></sup></sub></sup></sub></sup></sub></sup></sub></sup></sub></sup></sub></sup></sub></sup></sub></sup></sub></sup></sub></sup></sub></p>

**Comment by bkcsoft** [comment]
What if I'm in 2 organizations, where one is public and the other is private? Should my user be set to private or public? Users are non-exclusive members in a Organization, it doesn't make sense to have a hard link between a user and an organization IMO.

**Comment by camlafit** [comment]
Hi

In this case user could be present on explore list and only public organization could be listed on his profile.
If use override his profile to hide then shouldn't be displayed

Purpose is no to set individually each user state if organization is private

**Comment by agaida** [comment]
that would be ~~really nice to have~~ great.

**Comment by jscaux** [comment]
Hereby another vote in favour of having per-user public visibility settings in explore/users.

**Comment by camlafit** [comment]
Hello

as #1763 is merged, could be possible to finalize this issue ? Could be very nice to have a full private  gitea instance :)

Thanks again

**Comment by lunny** [comment]
@camlafit I think this one is not the same as #1763 ? This is for user and that's for organization.

**Comment by camlafit** [comment]
Yes is not same, but follows. Now we can set repo and organization as full private , miss user management part to be complete.

As discussed with @DblK was look easier to solve this issue only after to have close #1763 .

**Comment by lunny** [comment]
So let's keep this open
